Dockerfile

```diff
@@ -34,18 +34,20 @@ RUN --mount=type=cache,target=/root/.cache/go-build make sonicd sonictool
 # > docker run -e VALIDATOR_NUMBER=2 -e VALIDATORS_COUNT=5 -i -t sonic
 #
 FROM debian:bookworm
-COPY --from=client-build /client/build/sonicd .
-COPY --from=client-build /client/build/sonictool .
+COPY --from=client-build /client/build/sonic* .
```

Comment:
I would keep the old change and copy specific files to a container. This is to avoid copying of unwanted files.

Reply:
Now does `COPY --from=client-build /client/build/sonicd /client/build/sonictool .` to copy specific files using the same layer.